app/controllers/events_controller.rb

```diff
@@ -0,0 +1,37 @@
+class EventsController < ApplicationController
+  before_action :logged_in_user, only: [:new, :create]
+  def new
+    @event = Event.new
+  end
+
+  def create
+    unless current_user.try(:superadmin?)
```

Comment:
But the create action still has 6 lines
On the line:
 `redirect_to events_path and return false`
 if I use `&&` in place of `and` as code climate had suggested, it raises an error.